File: Java/AlgoExpert/Greedy Algorithms/Easy/Class Photos.java
import java.util.*;

class Program {

  public boolean classPhotos(
      ArrayList<Integer> redShirtHeights, ArrayList<Integer> blueShirtHeights) {
    // Write your code here.
		Collections.sort(redShirtHeights);
		Collections.sort(blueShirtHeights);
		boolean redShirtsBackRow = redShirtHeights.get(0) > blueShirtHeights.get(0);
		
		for (int i = 0; i < redShirtHeights.size(); i++) {
			int redShirtHeight = redShirtHeights.get(i);
			int blueShirtHeight = blueShirtHeights.get(i);
			
			if (redShirtsBackRow) {
				if (blueShirtHeight >= redShirtHeight) return false;
			} else {
				if (redShirtHeight >= blueShirtHeight) return false;
			}
		}
		
    return true;
  }
}

File: Python/AlgoExpert/Greedy Algorithms/Easy/classPhotos.py
def classPhotos(redShirtHeights, blueShirtHeights):
    # Write your code here.
    redShirtHeights.sort()
    blueShirtHeights.sort()

    red_shirt_taller = redShirtHeights[0] > blueShirtHeights[0]

    for i in range(len(redShirtHeights)):
        if red_shirt_taller:
            if redShirtHeights[i] <= blueShirtHeights[i]:
                return False
        else:
            if redShirtHeights[i] >= blueShirtHeights[i]:
                return False
    
    return True
